activities/member-relations/collect-information.md

```diff
@@ -0,0 +1,46 @@
+---
+type: Resource
+---
+
+# What information to collect about potential members for member relations 
```

Comment:
@clausmullie What do you think? It's more concise, and very clear that we start this early on for potential members.
```suggestion
# Potential member information collected by the Foundation
```